var/www/html/consul-templates/wp-config.php.ctmpl

```diff
@@ -71,13 +72,19 @@ define( 'WP_DEBUG_DISPLAY', {{key_or_default "wordpress/config/WP_DEBUG_DISPLAY"
  * Debug mode
  * Debugging? Enable these.
  */
-define( 'SAVEQUERIES', {{key_or_default "wordpress/config/SAVEQUERIES" "true" | parseInt}} );
-define( 'WP_DEBUG', {{key_or_default "wordpress/config/WP_DEBUG" "true" | parseInt}} );
+
+# turn on WP caching
+# TODO: template this with consul template to remove if no memcached servers are avaliable?
```

Comment:
You can move `define('WP_CACHE', true);` into `memcached-config.php.ctmpl` so that it can be turned on inside the `{{ if service "memcached" }}` conditional there.